Tools/Driver.Common.targets

```diff
@@ -0,0 +1,113 @@
+  4. $(Feature_PackOne) - Import Driver.PackOne.targets
+***********************************************************************************************
+-->
+<Project ToolVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+<!--
+look at https://msdn.microsoft.com/ru-ru/library/dd722601.aspx?f=255&MSPPError=-2147217396 for an example of TokenReplace task
+look at http://blogs.clariusconsulting.net/kzu/how-to-perform-regular-expression-based-replacements-on-files-with-msbuild/ for an example of RegexTransform task
+============================================================
+            SubstituteFileContent
+Replaces text in sources files with multiple substitutions.
+required Items: SourceFiles and Substituions (with Identity/ReplaceWith metadata)
+
+Example item:
+        <Substition Include="MACRO_VENDOR">
+            <ReplaceWith>Vendor Corp.</ReplaceWith>
+        </Substition>
+
+Invoking the task:
+    <SubstituteFileContent SourceFiles="@(SourceFile)" Substituions="@(Substition)" />
+============================================================
+-->
+  <UsingTask TaskName="SubstituteFileContent"
+             TaskFactory="CodeTaskFactory"
+             AssemblyFile="$(MSBuildToolsPath)\Microsoft.Build.Tasks.v4.0.dll">
+    <ParameterGroup>
+      <SourceFiles ParameterType="Microsoft.Build.Framework.ITaskItem[]" Required="true" />
+      <Substituions ParameterType="Microsoft.Build.Framework.ITaskItem[]" Required="true" />
+    </ParameterGroup>
+    <Task>
+      <Using Namespace="System.IO" />
+      <Using Namespace="System.Linq" />
+      <Using Namespace="System.Collections.Generic" />
+      <Using Namespace="System.Text.RegularExpressions" />
+      <Using Namespace="Microsoft.Build.Framework" />
+      <Code Type="Fragment" Language="cs">
+        <![CDATA[
+        var substitutions = new Dictionary<string,string>();
+        foreach (var item in Substituions) {
+            var replaceWhat = item.GetMetadata("Identity");
+            var replaceWith = item.GetMetadata("ReplaceWith") ?? "---";
+            substitutions.Add(replaceWhat,replaceWith);
+            Log.LogMessage(MessageImportance.High,"Adding {0}=>{1}",replaceWhat,replaceWith);
+        }
+
+        foreach (var item in SourceFiles) {
+            string fileName = item.GetMetadata("FullPath");
+            string fileNameOrig = fileName + ".orig";
+            string fileNameNew = fileName + ".new";
+            Log.LogMessage(MessageImportance.High, "Processing file {0}", fileName);
+
+            if(!File.Exists(fileName))
+            {
+                Log.LogMessage(MessageImportance.High, "Could not find file: {0}", fileName);
+                continue;
+            }
+
+            Log.LogMessage(MessageImportance.High, "Saving original context to: {0}", fileNameOrig);
+            File.Copy(fileName, fileNameOrig, true);
+
+            string content = File.ReadAllText(fileName);
```

Comment:
ReadLine will bring possible issues with the last empty line, /n/r format et.c.
As it recommended, for small files <1Mb it is better not to deal with that at all using ReadAllText